s.py

```diff
@@ -0,0 +1,229 @@
+import csv
+from dataclasses import dataclass
+from typing import Optional, List, Union, Dict
+
+import requests
+from bs4 import BeautifulSoup
+
+
+@dataclass
+class Everyone:
```

Comment:
This and `Developer` should be a single class. I suggest creating a folder named `model` and a file `developer.py` with this class declaration